controllers/cm_event_handler.go

```diff
@@ -0,0 +1,59 @@
+package controllers
```

Comment:
Earlier reconcile loop was called with CM?

Reply:
Yes. So had to use an explicit event handler. Similar to how node events are handled